src/main/kotlin/it/pagopa/wallet/eventdispatcher/service/WalletCDCService.kt

```diff
@@ -19,17 +18,36 @@ class WalletCDCService(
 
     private val log = LoggerFactory.getLogger(WalletCDCService::class.java.name)
 
-    fun sendToKafka(event: LoggingEvent): Mono<Unit> {
-        log.info("Sending CDC event to Kafka: [{}]", event.id)
+    fun sendToKafka(event: WalletLoggingEvent): Mono<Unit> {
+
+        log.info(
+            "Process event with id [{}] of type [{}] with walletId [{}] published on [{}]",
```

Comment:
I think we can remove this log since we already log inside both doOnSuccess and doOnError methods

Reply:
done here ae8064a3b197cd935027bcc1c03d40460a2207e3